### DISCUSSION TITLE
Nequip for orbnet dataset

### DISCUSSION THREAD

**Discussion by gianscarpe** [mir-group/nequip#423] [General]
Dear authors,
We're planning to train Nequip on Orbnet dataset (https://figshare.com/articles/dataset/OrbNet_Denali_Training_Data/14883867), which is shipped as ASE xyz files. First, I noticed you recompute energy through ASE calculator when dealing with ASE dataset (at https://github.com/mir-group/nequip/blob/9882562b32b89785e56c9e7681f0d3527d900568/nequip/data/AtomicData.py#L406). Is this the suggested way to proceed, or can we adopt the GT energy provided by the dataset?
Second, I've a general question about the "per_species_rescale_scales" and  "per_species_rescale_shifts"  parameters. The default value is `dataset_forces_rms` but, as orbnet provides only energy labels, we changed to `dataset_per_atom_total_energy_std` and `dataset_per_atom_total_energy_mean`. Can you briefly explaining what is the purpose of this parameter?
Thank you very much for your time!

**Comment by Linux-cpp-lisp** [comment]
Hi @gianscarpe ,

Thanks for your interest in our code!

This is a consequence of how ASE reads in energy and force data.  It should work fine with your data as long as it is in the standard `energy` key.  What do you mean by `GT energy`?

> we changed to dataset_per_atom_total_energy_std and dataset_per_atom_total_energy_mean

Probably the easiest thing (especially on the latest `develop`, which you should make sure to pull and use) is to set the scales to `null` (as those config files do) and the shifts as you did to `dataset_per_atom_total_energy_mean` or the true DFT isolated atom energies.
